x-pack/plugins/rollup/public/crud_app/services/humanized_numbers.js

```diff
@@ -41,21 +41,30 @@ export function getOrdinalValue(number) {
   //   values: { number },
   // });
 
-  let lastDigit = number.toString().substr(-1);
+  const num = number && number.toString();
```

Comment:
Protects against falsey (including `0`) values

Reply:
Can you add this as a comment in the code?